src/feed/feed.service.ts

```
@@ -67,6 +74,19 @@ export class FeedService {
     }));
   }
 
+  async moderateJob(jobId: string, status: 'approved' | 'rejected'): Promise<Job> {
```

ESLint

[error] Replace `jobId:·string,·status:·'approved'·|·'rejected'` with `⏎····jobId:·string,⏎····status:·'approved'·|·'rejected',⏎··`

(prettier/prettier)